doc/sphinx-guides/source/index.rst

```diff
@@ -15,6 +15,7 @@ These documentation guides are for the |version| version of Dataverse. To find g
 
   user/index
   admin/index
+  ai/index
```

Comment:
I put the AI Guide third, after User and Admin. But maybe it should be second, after User? 🤔